packages/shorebird_code_push_protocol/lib/src/models/release_patch.dart

```diff
@@ -41,6 +42,16 @@ class ReleasePatch extends Equatable {
   /// Whether the patch has been rolled back.
   final bool isRolledBack;
 
+  /// The notes associated with the patch.
+  final String? notes;
```

Comment:
Can these notes be markdown? Might be good to elaborate a bit in the docs.

Reply:
Done.